CMakeLists.txt

```diff
@@ -1301,6 +1302,9 @@ endif()
 target_include_directories(mixxx-lib SYSTEM PUBLIC lib/fidlib)
 target_link_libraries(mixxx-lib PUBLIC fidlib)
 
+#KeyFinder
```

Comment:
This doesn't seem restrict to any particular version?

Reply:
Yes, this was just a quick hack to get it to build. I can add a proper CMake module to search for a pkgconfig file. I just added a pkgconfig file to https://github.com/ibsh/libKeyFinder/pull/33

Reply:
done in 3188119 and added check for library version >= 2